elements/rh-alert/demo/rh-alert.js

```diff
@@ -3,5 +3,10 @@ import '@rhds/elements/rh-alert/rh-alert.js';
 
 const root = document.querySelector('[data-demo="rh-alert"]')?.shadowRoot ?? document;
 
-root.querySelector('rh-alert');
+const dismissiableAlerts = root.querySelectorAll('rh-alert[dismissable]');
+
+dismissiableAlerts.forEach(alert => {
+  // eslint-disable-next-line no-console
+  alert.addEventListener('close', () => console.log('close'));
```

Comment:
Let's leave one example in the demo where the default is prevented, to be sure that's possible, maybe run a timer and a spinner and then close to demo async workflows.

Reply:
Added